packages/cloud-cognitive/src/global/js/hooks/useResetCreateComponent.js

```diff
@@ -35,7 +38,8 @@ export const useResetCreateComponent = ({
         Number(initialStep) <= Number(totalSteps) &&
         Number(initialStep) > 0
       ) {
-        setCurrentStep(Number(initialStep));
+        let numberOfHiddenSteps = getNumberOfHiddenSteps(stepData, initialStep);
```

Comment:
Change this `let` to `const` since it doesn't need to be reassigned

```suggestion
        const numberOfHiddenSteps = getNumberOfHiddenSteps(stepData, initialStep);
```